test/upload_validation_test.ts

```diff
@@ -40,12 +40,15 @@ import {validateSyncTableSchema} from '../testing/upload_validation';
 import {validateVariousAuthenticationMetadata} from '../testing/upload_validation';
 
 describe('Pack metadata Validation', () => {
-  async function validateJson(obj: Record<string, any>) {
-    return validatePackVersionMetadata(obj);
+  async function validateJson(obj: Record<string, any>, sdkVersion?: string) {
+    const packageJson = await import('../package.json');
```

Comment:
I think this is what's tripping up the CI. One of the build tools is too smart for its own good and as far as I can tell sniffs the dynamic imports for string literals, and includes the imported literal paths in the compiled output in `dist`.

The other places where we dynamically import this (intentionally) don't use string literals in part because of this. If you change this line to something like `await import('' + '../package.json');` (and then rebuild) it should avoid the issue. 😱